average.js

```diff
@@ -1,6 +1,9 @@
 
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+  const filterFunc = (element) => !(isNaN(element));
+  
+  let arr = numbers.filter(filterFunc)
```

Comment:
Try a more expressive name in the place of `arr`. Variable names are meant to convey their intent without having to read the whole code.